platforms/substrate/configuration/roles/create/certificates/ambassador/tasks/main.yaml

```diff
@@ -39,7 +39,7 @@
 # Gets the existing ambassador tls certs
 - name: Get ambassador and tls certs from Vault
   shell: |
-    vault kv get -format=yaml {{ vault.secret_path | default('secretsv2') }}/{{ organisation }}/tlscerts
+    vault kv get -format=yaml {{ vault.secret_path | default('secretsv2') }}/{{ network.env.type }}/{{ organisation }}/tlscerts
```

Comment:
Should we be using ansible k8s module and get the certs from secrets, instead of running vault commands ?

Reply:
Updated to utilize the K8s module to check if the secret is already present. If not, then it will create one. Thank you.